services/app-api/src/resolvers/questionResponse/createQuestion.ts

```diff
@@ -126,6 +128,40 @@ export function createQuestionResolver(
             })
         }
 
+        let stateAnalystsEmails =
+            await emailParameterStore.getStateAnalystsEmails(
+                contractResult.stateCode
+            )
+        //If error log it and set stateAnalystsEmails to empty string as to not interrupt the emails.
+        if (stateAnalystsEmails instanceof Error) {
+            logError('getStateAnalystsEmails', stateAnalystsEmails.message)
+            setErrorAttributesOnActiveSpan(stateAnalystsEmails.message, span)
+            stateAnalystsEmails = []
+        }
+
+        const roundNumber = questionResult.responses.length + 1
+        const sendQuestionsCMSEmailResult = await emailer.sendQuestionsCMSEmail(
+            contractResult.revisions[0],
+            stateAnalystsEmails,
+            user,
+            statePrograms,
+            dateAsked,
+            roundNumber
+        )
+
+        if (sendQuestionsCMSEmailResult instanceof Error) {
+            logError(
+                'sendQuestionsCMSEmail - CMS email failed',
+                sendQuestionsCMSEmailResult
+            )
+            setErrorAttributesOnActiveSpan('CMS email failed', span)
```

Comment:
Let's add the contract ID and Question ID to this email error message for debugging if this error was ever logged.